MSMetaEnhancer/libs/services/IDSM.py

```diff
@@ -26,11 +26,11 @@ def __init__(self, session):
                            {'code': 'isomeric_smiles', 'label': 'CHEMINF_000379'}]
 
         # generate top level methods defining allowed conversions
-        conversions = [('name', 'inchi', 'from_name'),
-                       ('name', 'iupac_name', 'from_name'),
-                       ('name', 'formula', 'from_name'),
-                       ('name', 'canonical_smiles', 'from_name'),
-                       ('name', 'isomeric_smiles', 'from_name'),
+        conversions = [('compound_name', 'inchi', 'from_name'),
+                       ('compound_name', 'iupac_name', 'from_name'),
+                       ('compound_name', 'formula', 'from_name'),
+                       ('compound_name', 'canonical_smiles', 'from_name'),
+                       ('compound_name', 'isomeric_smiles', 'from_name'),
```

Comment:
Since this is the canonical form in matchms now, don't we have to change this for all services?

Reply:
That is a good point! Thank you, fixed in [40ec3d2](https://github.com/RECETOX/MSMetaEnhancer/pull/77/commits/40ec3d2dbd3ff9d8f137755d7917983efb774d5d).